src/app/components/planner-list/planner-list.component.less

```diff
@@ -199,6 +199,11 @@ button.btn.btn-link.dropdown-toggle:focus {
   outline: none;
 }
 
+.planner-hack-title-truncate {
```

Comment:
Why would you name it to hack? Is it really a hack or the correct way to do it? It seems elegant to me.

Reply:
I think it's a hack. If it's the elegant way of doing it. I will rename it.